javacutil/src/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -331,6 +331,22 @@ public static boolean containsSameByClass(
     }
 
     /**
+     * Returns the AnnotationMirror in {@code c} that has the same class as {@code anno}.
+     *
+     * @return AnnotationMirror with the same class as {@code anno} iff c contains anno, according
+     * to areSameByClass; otherwise, {@code null}
+     */
+    public static AnnotationMirror getAnnotationByClass(
```

Comment:
To reduce duplication, do you want to change `containsSameByClass` to simply `return getAnnotationByClass(c, anno) != null;"?